go-controller/pkg/node/node.go

```diff
@@ -99,6 +99,68 @@ func setupOVNNode(node *kapi.Node) error {
 			return fmt.Errorf("error setting OVS encap-port: %v\n  %q", errSet, stderr)
 		}
 	}
+	if config.Monitoring.NetFlowTargets != nil {
+		collectors := ""
+		for _, v := range config.Monitoring.NetFlowTargets {
+			collectors += "\"" + v.Host.String() + ":" + strconv.Itoa(v.Port) + "\"" + ","
+		}
+		collectors = strings.TrimSuffix(collectors, ",")
+
+		_, stderr, err := util.RunOVSVsctl(
+			"--",
+			"--id=@netflow",
+			"create",
+			"netflow",
+			fmt.Sprintf("targets=[%s]", collectors),
+			"active_timeout=60",
+			"--",
+			"set", "bridge", "br-int", "netflow=@netflow",
+		)
+		if err != nil {
+			return fmt.Errorf("error setting NetFlow: %v\n  %q", err, stderr)
+		}
+	}
+	if config.Monitoring.SFlowTargets != nil {
+		collectors := ""
+		for _, v := range config.Monitoring.SFlowTargets {
+			collectors += "\"" + v.Host.String() + ":" + strconv.Itoa(v.Port) + "\"" + ","
+		}
+		collectors = strings.TrimSuffix(collectors, ",")
+
+		_, stderr, err := util.RunOVSVsctl(
+			"--",
+			"--id=@sflow",
+			"create",
+			"sflow",
+			"agent=ovn-k8s-mp0",
```

Comment:
what's this for? please use constant

Reply:
sflow requires to specify the interface that is capturing, since this is br-int that's why I use this port.